### ISSUE TITLE
1.13.2

### ISSUE THREAD

**Issue by random1167** [utarwyn/SuperJukebox#7]
```
No enum constant org.bukkit.Sound.BLOCK_NOTE_HARP
        at java.lang.Enum.valueOf(Enum.java:238) ~[?:1.8.0_191]
        at org.bukkit.Sound.valueOf(Sound.java:1) ~[spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at fr.utarwyn.superjukebox.music.model.Instrument.getSound(Instrument.java:61) ~[SuperJukebox-1.1.1.jar:?]
        at fr.utarwyn.superjukebox.music.MusicPlayer.playTick(MusicPlayer.java:128) ~[SuperJukebox-1.1.1.jar:?]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:391) [spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:888) [spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:830) [spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:728) [spigot-1.13.2.jar:git-Spigot-82e117e-2b2d084]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]

```
LOG:
https://hastebin.com/asunevivaf.cs
https://hastebin.com/raw/asunevivaf - Text only

**Comment by TheCoolOne123** [comment]
1. You're on 1.13 right?
If you are, 1.13+ is not supported. This is because the noteblock enum (Where the noteblock sounds come from) is changed in 1.13

**Comment by utarwyn** [comment]
This bug should be fixed with the commit 7791d6e5f41e35bbc9388a8e933cdd4ec272eb90.
There is another **bug on 1.13 servers with particles**. I will work on that asap 😄

**Comment by utarwyn** [comment]
Should be fixed with the version **0.2.0** of the plugin, which is available here : https://github.com/utarwyn/SuperJukebox/releases/tag/v0.2.0

Don't hesitate to open a new issue or contact us on Discord if you have any issue with the plugin.
